app/models/responses/DowntimeResponse.scala

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models.responses
+
+import models.responses.ErrorResponse.DowntimeResponseError
+import models.{DowntimeChannel, MongoDateTimeFormats}
+import org.slf4j.LoggerFactory
+import play.api.http.Status.OK
+import play.api.libs.functional.syntax._
+import play.api.libs.json._
+import uk.gov.hmrc.http.{HttpReads, HttpResponse}
+
+import java.time.LocalDateTime
+
+case class Downtime(affectedChannel: DowntimeChannel, start: LocalDateTime, end: LocalDateTime)
+
+object Downtime {
+
+  implicit val writes: Writes[Downtime] = {
+    (
+      (__ \ "affectedChannel").write[DowntimeChannel](DowntimeChannel.format) and
+        (__ \ "start").write(MongoDateTimeFormats.localDateTimeWrite) and
+        (__ \ "end").write(MongoDateTimeFormats.localDateTimeWrite)
+      ) (unlift(Downtime.unapply))
+  }
```

Comment:
This is not needed. You can change this below

**implicit val format: OFormat[DowntimeResponse] = Json.format[DowntimeResponse]**

to 

**implicit val reads: Reads[DowntimeResponse] = Json.reads[DowntimeResponse]**

Reply:
the writes is being used on the integration tests when we do:
Json.toJson(DowntimeResponse(
          Seq(
            Downtime(GBDepartures, LocalDateTime.now().minusHours(1), LocalDateTime.now()),
            Downtime(XIDepartures, LocalDateTime.now().minusHours(1), LocalDateTime.now()),
            Downtime(GBArrivals, LocalDateTime.now().minusHours(1), LocalDateTime.now()),
            Downtime(XIArrivals, LocalDateTime.now().minusHours(1), LocalDateTime.now()),
            Downtime(Web, LocalDateTime.now().minusHours(1), LocalDateTime.now()),
            Downtime(XML, LocalDateTime.now().minusHours(1), LocalDateTime.now())
          ),
          LocalDateTime.now()
        ))

Reply:
OK but if they are only used for ITs then the writes should really be in the ITs.

Reply:
Since Downtime is used in DowntimeResponse model, when we try to convert DowntimeResponse to Json, I believe it looks for an implicit converter in the Downtime model. I tried moving only the writes to the IT test file, but it doesnt seem to find it